### DISCUSSION TITLE
Password alteration

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8715] [Q&A] [answered]
Hello!

There is a PCI DSS requirement that every time users change the password, it cannot be the same as the previous four. Does Zitadel support this feature?

Thank you.

**Comment by mffap** [comment, marked as answer]
Hi @ashep this seems to be a duplicate of https://github.com/zitadel/zitadel/issues/8034

At the moment ZITADEL doesn't support this requirement unfortunately. We've opened now a summary issue to track ZITADEL's progress to a PCI DSS compliant IdP: https://github.com/zitadel/zitadel/issues/8716
Feel free to add any points we might be missing.
